### PR TITLE
[EDU-4317] feat: add Gatsby Blog Starter Kit guide

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-4317]

### Changes

- Add Gatsby Blog Starter Kit guide - EN and PT.
- Add links to Guides page.

### Additional links

n/a.

[EDU-4317]: https://aziontech.atlassian.net/browse/EDU-4317?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    with:
+        ....
+        azionPersonalToken: ${{ secrets.AZION_PERSONAL_TOKEN }}
+        ....
+
+```
+
+3. Abra um pull request com as alterações na main branch e inicie a implantação automática.
+
+Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações no aplicativo ou no repositório.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx:140]: aplicativo ou aplicação?

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+### Adicione um domínio personalizado
+
+A edge application criada tem um domínio Azion atribuído para torná-la acessível através do navegador. O domínio tem o seguinte formato: `xxxxxxxxxx.map.azionedge.net`. No entanto, você pode adicionar um domínio personalizado para que os usuários acessem sua aplicação por meio dele.
+
+import Button from '~/components/Button.astro';
+
+<Button href="/pt-br/documentacao/produtos/build/edge-application/domains/" text="saiba mais sobre domínios" variant="secondary" />
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx:148]: EN leva para um guia

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx (hunk 1 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx:140]:
```suggestion
Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações na aplicação ou no repositório.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx:148]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/configurar-dominio/" text="saiba mais sobre domínios" variant="secondary" /> 
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/templates/gatsby-blog-starter-kit.mdx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+:::
+
+- You must also activate:
+<Badge>
+[Edge Functions](/en/documentation/products/build/edge-application/edge-functions/)
+</Badge>
+<br/>
+<Badge>
+[Image Processor](/en/documentation/products/build/edge-application/image-processor//)
+</Badge>
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/gatsby-blog-starter-kit.mdx:42]:
```suggestion
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+:::
+
+Você também deve ativar:
+<Badge>
+[Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/)
+</Badge>
+<br/>
+<Badge>
+[Image Processor](/pt-br/documentacao/produtos/build/edge-application/image-processor/)
+</Badge>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx:42]:
```suggestion
```

**Pull request merged**